pages/report.tsx

```diff
@@ -75,15 +77,44 @@ function useInitialFormValues(): UnsubmittedFormResponse {
     email,
     should_subscribe: false,
     confirmed_get_app_updates: false,
+    image_uuid: formUuid,
   };
 }
 
 export default function Report() {
   const [currentStep, setCurrentStep] = useState(FormStep.SightingDetails);
   const { session } = useSessionUser();
-  const defaultFormFormValues = useInitialFormValues();
+  const [formUUID] = useState(uuidv4());
+  const defaultFormFormValues = useInitialFormValues(formUUID);
```

Comment:
This is just a roundabout way of instantiating the initial form values with an `image_id`. If we want to maintain an invariant where the field is clear if there are no associated images, we should actually use the state setter and null out the value if the user clears all files from the input! Otherwise, it would be cleaner to initialize the property in `useInitialFormValues` and leave it at that.